src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -167,21 +252,20 @@ class UniversalNavbar extends React.Component {
                   <NavLink href={logoHref} LinkComponent={LinkComponent}>
                     {LogoNotAnimated({ className: styles.logo })}
                   </NavLink>
-                  {renderDesktopLink(LINKS.NAVLINKS[0])}
-                  {renderDesktopLink(LINKS.NAVLINKS[1])}
+                  {renderTextLink(LINKS.NAVLINKS[0])}
```

Comment:
Yeah, again, I'd prefer us to utilize AsyncTypehead for these if possible.

Reply:
See https://github.com/getethos/ethos/pull/2167

Reply:
I'll have a look!

Reply:
Ok, update...chatted w/Cole and this is literally a navbar with a search icon that acts as a link:

![image](https://user-images.githubusercontent.com/142403/73476554-5372e880-4347-11ea-93f0-4d5ecec5221f.png)